content/functions/cond.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "cond"
+date: 2017-09-08
+description: "Return one of two arguments, depending on the value of a third argument."
```

Comment:
@jetwash 

Thanks for putting this together. How about the following for `description` instead?

> Returns one of two arguments according to the truthiness of a condition. 

Or

> Assigns one of two values to a variable according to the truthiness of a condition.

Still could be room for improvement, but the current description feels like it could apply to more than just `cond`.